packages/backend/src/openapi/exchange-rates.yaml

```diff
@@ -0,0 +1,59 @@
+openapi: 3.1.0
```

Comment:
will validation using this spec be added later when receiving the response from the /rates endpoint?

Reply:
I started adding it but then I realized that responses are already validated. See [here](https://github.com/interledger/rafiki/blob/main/packages/backend/src/rates/service.ts#L115). Hence, the spec is rather for documentation than for validation.